testing/simapp/app.go

```diff
@@ -319,40 +326,45 @@ func NewSimApp(
 
 	// SDK module keepers
 
-	app.AccountKeeper = authkeeper.NewAccountKeeper(
-		appCodec, keys[authtypes.StoreKey], app.GetSubspace(authtypes.ModuleName), authtypes.ProtoBaseAccount, maccPerms, sdk.GetConfig().GetBech32AccountAddrPrefix(),
-	)
+	app.AccountKeeper = authkeeper.NewAccountKeeper(appCodec, keys[authtypes.StoreKey], authtypes.ProtoBaseAccount, maccPerms, sdk.Bech32MainPrefix, authtypes.NewModuleAddress(govtypes.ModuleName).String())
+
 	app.BankKeeper = bankkeeper.NewBaseKeeper(
-		appCodec, keys[banktypes.StoreKey], app.AccountKeeper, app.GetSubspace(banktypes.ModuleName), app.ModuleAccountAddrs(),
+		appCodec,
+		keys[banktypes.StoreKey],
+		app.AccountKeeper,
+		BlockedAddresses(),
```

Comment:
Is there a reason why we wouldn't be able to use `app.ModuleAccountAddrs()` here instead of the new function `BlockedAddresses()`? Because in `app.ModuleAccountAddrs` we take out the `ibcmock` module from the list of blocked addresses, but since that is not done in `BlockedAddresses`, I think that's causing [this test to fail](https://github.com/cosmos/ibc-go/actions/runs/3674175244/jobs/6212090001).

Basically my question is: do we need to have a new function `BlockedAddresses` or can we still use the existing one `app.ModuleAccountAddrs` we had?

Reply:
Yes lets use a `BlockedAddresses` function. It is a hack that `ModuleAccountAddrs()` doesn't return the mock module address. We should delete the mock module from the addresses returned in `BlockedAddresses` like how it is done for gov:
```go
delete(modAccAddrs, authtypes.NewModuleAddress(ibcmock.ModuleName).String())
```